README.md

```diff
@@ -31,8 +31,12 @@ Mandatory:
 - binutils, cpio, gzip, bzip, xz and zstd
 
 Optional:
+- checkbashisms
```

Comment:
This is part of `devscripts` in most distributions.

Reply:
Replace this with devscripts, or "devscripts, or some package that provides checkbashisms" ?

Reply:
Please use `devscripts`.